pandas/tseries/tests/test_period.py

```diff
@@ -2537,6 +2540,16 @@ def test_searchsorted(self):
             ValueError, 'Different period frequency: H',
             lambda: pidx.searchsorted(pd.Period('2014-01-01', freq='H')))
 
+    def test_round_trip_pickle(self):
+
+        p = Period('2000Q1')
+        pickle_path = os.path.join(tm.get_data_path(),
+                                   'period.pickle')
+
+        with open(pickle_path, 'wb') as f: pickle.dump(p, f)
```

Comment:
https://github.com/pydata/pandas/blob/master/pandas/io/tests/generate_legacy_storage_files.py

u may need to rebase on master
this was recently changed